perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -2543,6 +2557,20 @@ public void updateFormItem(PerunSession sess, ApplicationFormItem item) throws P
 				}
 
 			}
+
+			// if pending vo application data can be used for group application, overwrite value with it
+			for (ApplicationFormItemData appFormItemData : pendingVoApplicationData) {
+				String vosAppDestinationAttribute = appFormItemData.getFormItem().getPerunDestinationAttribute();
+				// remove password field
+				if (itemW.getFormItem().getType() == PASSWORD) {
+					it.remove();
+					continue;
+				}
```

Comment:
I'm not sure if this should be here. This would probably remove all password fields from all applications if the vo application has been submitted. I think you should remove all of this code.

Reply:
You're right, and the previous checks should cover the necessary cases when we want to remove the password. Removed this piece of code.